### PR TITLE
new cal csvs

### PR DESCRIPTION
**Pull request by rtravis-whoi**
*note for the ADCPs - cert of conformity date has been used for cal csv dates (exception to cal csv procedure info)

### CODE REVIEW

File: calibration/ADCPSL/CGINS-ADCPSL-20502__20230619.csv [hunk 1]
```diff
@@ -0,0 +1,8 @@
+serial,name,value,notes
+20502,CC_scale_factor1,0.45,constant; source_fiile ADCPS-L_WHLS75-1500-I_SN_20502_Configuration_Summary_2023-06-19.pdf
```
Comment (cooleyky) [inline calibration/ADCPSL/CGINS-ADCPSL-20502__20230619.csv:2]: Extra "i" in "source_file" on this line

File: calibration/ADCPTF/CGINS-ADCPTF-22672__20230724.csv [hunk 2]
```diff
@@ -0,0 +1,5 @@
+serial,name,value,notes
+22672,CC_scale_factor1,0.45,source_file ADCPT-F_WHS150-1500_SN_22672_Configuration_Summary_2023-07-24.pdf
+22672,CC_scale_factor2,0.45,
+22672,CC_scale_factor3,0.45,
+22672,CC_scale_factor4,0.45,
```
Comment (cooleyky) [inline calibration/ADCPTF/CGINS-ADCPTF-22672__20230724.csv:5]: looks good

File: calibration/ADCPTG/CGINS-ADCPTG-24543__20230726.csv [hunk 3]
```diff
@@ -0,0 +1,8 @@
+serial,name,value,notes
+24543,CC_scale_factor1,0.45,constant; source_file ADCPT-G_WHS150-1500-I_SN_24543_Configuration_Summary_2023-07-26.pdf
+24543,CC_scale_factor2,0.45,constant
+24543,CC_scale_factor3,0.45,constant
+24543,CC_scale_factor4,0.45,constant
+24543,CC_bin_size,0,in cm; return from vendor - not allocated yet
+24543,CC_dist_first_bin,0,in cm; return from vendor - not allocated yet
+24543,CC_orientation,1,1 is upward looking; 0 is downward looking
```
Comment (cooleyky) [inline calibration/ADCPTG/CGINS-ADCPTG-24543__20230726.csv:8]: looks good

File: calibration/PCO2WC/CGINS-PCO2WC-C0120__20230614.csv [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+C0120,CC_cala,0.0505,source_file PCO2W-C_SAMI-CO2_SN_C0120_Calibration_Files_2023-06-14.zip
+C0120,CC_calb,0.5967,
+C0120,CC_calc,-1.4827,
+C0120,CC_calt,10.8529,
+C0120,CC_ea434,19706,constant
+C0120,CC_ea620,34,constant
+C0120,CC_eb434,3073,constant
+C0120,CC_eb620,44327,constant
+C0120,CC_sami_bits,14,
+C0120,CC_cal_range,"[201,997]",
```
Comment (cooleyky) [inline calibration/PCO2WC/CGINS-PCO2WC-C0120__20230614.csv:11]: looks good

**Review by cooleyky — COMMENTED** [review]
Everything looks good except for a typo in the notes for ADCPS-L 20502.

File: calibration/ADCPSL/CGINS-ADCPSL-20502__20230619.csv [hunk 5]
```diff
@@ -1,5 +1,5 @@
 serial,name,value,notes
-20502,CC_scale_factor1,0.45,constant; source_fiile ADCPS-L_WHLS75-1500-I_SN_20502_Configuration_Summary_2023-06-19.pdf
+20502,CC_scale_factor1,0.45,constant; source_file ADCPS-L_WHLS75-1500-I_SN_20502_Configuration_Summary_2023-06-19.pdf
 20502,CC_scale_factor2,0.45,constant
 20502,CC_scale_factor3,0.45,constant
 20502,CC_scale_factor4,0.45,constant
```
Comment (cooleyky) [inline calibration/ADCPSL/CGINS-ADCPSL-20502__20230619.csv:5]: Looks good now!

**Review by cooleyky — APPROVED** [review]
Looks good

**Review by reedan88 — APPROVED** [review]
Looks good.

**Pull request merged**